WalletWasabi/Services/MempoolBehavior.cs

```diff
@@ -55,20 +55,20 @@ private async void AttachedNode_MessageReceivedAsync(Node node, IncomingMessage
 			}
 			catch (OperationCanceledException ex)
 			{
-				Logger.LogDebug<MempoolBehavior>(ex);
+				Logger.LogDebug(ex);
 			}
 			catch (Exception ex)
 			{
-				Logger.LogInfo<MempoolBehavior>($"Ignoring {ex.GetType()}: {ex.Message}");
-				Logger.LogDebug<MempoolBehavior>(ex);
+				Logger.LogInfo($"Ignoring {ex.GetType()}: {ex.Message}");
```

Comment:
Should this kind of log entries end with a period too?

Reply:
I think so but that would be too much and it is out of the scope of this PR.